packages/e2e-tests/test-applications/create-react-app/src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import logo from './logo.svg';
+import * as Sentry from '@sentry/react';
+import './App.css';
+
+function App() {
+  return (
+    <Sentry.ErrorBoundary fallback={<p>An error has occurred</p>}>
```

Comment:
Let's make us of the more fleshed out fallback.

```suggestion
    <Sentry.ErrorBoundary
        fallback={({ error, componentStack, resetError }) => (
          <React.Fragment>
            <div>You have encountered an error</div>
            <div>{error.toString()}</div>
            <div>{componentStack}</div>
            <button
              onClick={() => {
                this.setState({ message: "This is my app" });
                {/* When resetError() is called it will remove the Fallback component */}
                {/* and render the Sentry ErrorBoundary's children in their initial state */}
                resetError();
              }}
            >
              Click here to reset!
            </button>
          </React.Fragment>
        )}
      >
```